README.md

```diff
@@ -8,6 +8,11 @@ It is written in [Go][go-site] and is a soft fork of [Hugo][hugo-site].
 [Twitter](https://twitter.com/GothamHQ_)
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/gothamhq/gotham)](https://goreportcard.com/report/github.com/gothamhq/gotham)
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/gothamhq/gotham)
+[![CircleCI](https://circleci.com/gh/gothamhq/gotham.svg?style=shield)](https://circleci.com/gh/gothamhq/gotham)
+[![codecov](https://codecov.io/gh/gothamhq/gotham/branch/master/graph/badge.svg)](https://codecov.io/gh/gothamhq/gotham)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
```

Comment:
What's the point of this badge?

Reply:
I add a few that I like looks at similar projects. The PRs Welcome badge is there to hopefully motivate people to contribute.

I'm open to edit / add / remove any of these.

Do you want this one removed?
Are the others you would want to include?

Reply:
We say they're welcome in the readme. I tend to view badges as indicators for information not available on the page already, with the exception of a license.

I'm not a fan but it's not the end of the world for me either. Keep it if you like it.